### DISCUSSION TITLE
laser-eye-correction

### DISCUSSION THREAD

**Discussion by giscus** [mrocklin/mrocklin.github.io#10] [Giscus]
# laser-eye-correction

This article records my experience going through laser corrective eye surgery. Typically folks get LASIK. I got PRK/ASA instead, which is slightly more involved. This will go through the following ...

https://matthewrocklin.com/laser-eye-correction.html

**Comment by andersy005** [comment]
Thank you for sharing your experience, Matt! I found it very informative and helpful. I have been considering laser corrective eye surgery for a while and your post gave me more confidence to go ahead with it.

I’m glad to hear that you had a good outcome and no regrets. I hope your vision continues to improve and you enjoy your new glasses-free life! 😊

**Comment by jacobtomlinson** [comment]
Thanks for writing this up, I also found it really interesting. Glad things are improving for you. 

I'm curious what the doctor/optometrist said about how things will change over time. Eyes get worse with age right? I didn't need glasses until my mid-twenties and every couple of years they bump my prescription up a little. If you have laser corrective surgery does that just zero you out again? Will you at some point need glasses/surgery again?

**Reply by mrocklin** [reply]
*I am not an optometrist*

If your prescription is changing meaningfully then no, you may not want to get corrected.  Your vision may continue to change.  

Typically myopia is caused by your cornea shaping itself oddly.  In most people this develops when you're young and then stabilizes in adulthood.  This is what laser eye correction corrects.

Additionally as you turn 40 or so your lens stiffens, which is why older folks need reading glasses (hard to dynamically shape the lens to focus up close).  This is unrelated to laser eye correction.

**Reply by jacobtomlinson** [reply]
Interesting! Thanks :)
